libs/executors/src/exception_list_callbacks.cpp

```diff
@@ -0,0 +1,31 @@
+//  Copyright (c) 2020      ETH Zurich
+//                2007-2017 Hartmut Kaiser
+//                2017-2018 Taeguk Kwon
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#include <hpx/config.hpp>
+#include <hpx/executors/exception_list.hpp>
+
+namespace hpx { namespace parallel { inline namespace v1 { namespace detail {
+    static exception_list_termination_handler_type
+        exception_list_termination_handler_f;
```

Comment:
Same here, function local statics might be safer.

Reply:
Yeah, I just updated both files to use function statics.